rdr_service/model/rex.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy import Column, Integer, String
+
+from rdr_service.model.base import RexBase
+
+
+class TestModel(RexBase):
+
+    __tablename__ = "test"
+
+    id = Column("id", Integer, primary_key=True)
+    name = Column("name", String(256), nullable=False)
+    place = Column("place", String(256), nullable=True)
```

Comment:
I appreciate seeing that we're able to generate migrations for the new databases. But do we need to have these test classes and their migration files be merged? It seems like we're setting ourselves up to need to create more migrations to remove the tables later, and we'll always have the test table in the migration files.

Reply:
I'm still leaning towards removing the Test class and migrations. I don't think git would keep the alembic/nph and /rex folders, but we can recreate them when we set up the initial migrations (if alembic doesn't create them on its own).

Reply:
I removed the `test` table migrations